cmd/zarb/tx/common.go

```diff
@@ -0,0 +1,36 @@
+package tx
+
+import (
+	"strings"
+
+	"github.com/zarbchain/zarb-go/cmd"
+	"github.com/zarbchain/zarb-go/keystore/key"
+	"github.com/zarbchain/zarb-go/tx"
+	zarb "github.com/zarbchain/zarb-go/www/grpc/proto"
+)
+
+func signAndPublish(trx *tx.Tx, keyfile, auth string, grpcClient zarb.ZarbClient) {
+	//sign transaction
+	k, err := key.DecryptKeyFile(keyfile, auth)
+	if err != nil {
+		cmd.PrintErrorMsg("Couldn't retrieve Key: %v", err)
+		return
+	}
+	k.ToSigner().SignMsg(trx)
+
+	cmd.PrintWarnMsg("you are about to publish:")
+	cmd.PrintJSONObject(trx)
+	confirm := cmd.PromptInput("are you sure [y/n]:")
```

Comment:
```
confirm := cmd.PromptInput("The operation is not reversible. Are you sure? [yes/no]")
	if !strings.HasPrefix(strings.ToLower(confirm), "yes") {
```

Reply:
This is really important. Let user write `yes`. Make sure not accidentally press a `y` key.